azure-pipelines.yml

```diff
@@ -10,10 +10,14 @@ resources:
       type: github
       endpoint: github
       name: asottile/azure-pipeline-templates
-      ref: refs/tags/v1.0.0
+      ref: refs/tags/v2.0.0
 
 jobs:
 - template: job--pre-commit.yml@asottile
+- template: job--python-tox.yml@asottile
+  parameters:
+    toxenvs: [py37]
+    os: [windows, macos]
```

Comment:
no macos please, I gave you exactly what to copy paste!

Reply:
Just out of curiosity, why don't you want mac OS? Is it an azure billing thing or how many concurrent workers you get (i.e. on github actions it is only 5 concurrent mac workers, while in total you get 20)?

Reply:
waste of time and resources